projects/storefrontstyles/scss/components/myaccount/order-history/_order-return-request-list.scss

```diff
@@ -82,6 +82,10 @@
 
   .cx-order-history-code {
     text-decoration: underline;
+
+    @include media-breakpoint-down(sm) {
```

Comment:
I'd prefer combining those selectors in a single media queries: 

```scss
@include media-breakpoint-down(sm) {
  .cx-order-history-thead-mobile,
  .cx-order-history-code {
    display: none;
  }
  .cx-order-history-placed,
  .cx-order-history-status,
  .cx-order-history-total {
    text-align: left;
  }
}
```

but we can postpone that if you like

Reply:
so i combined only the last block, because `.cx-order-history-code` doesn't have a `display:none;`